<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1310</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(185, 217, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="logsText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>651</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Lucida Sans&quot;;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
    <property name="centerOnScroll">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>750</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 75 16pt &quot;Nirmala UI&quot;;</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>750</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 75 16pt &quot;Nirmala UI&quot;;</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="tableText">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>330</y>
      <width>581</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Lucida Sans&quot;;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
    <property name="centerOnScroll">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="sendFirstButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>660</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Lucida Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Send first</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="sendAllButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>710</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Lucida Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Send all entries</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>660</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Lucida Sans&quot;;</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>660</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Lucida Sans&quot;;</string>
    </property>
    <property name="text">
     <string>entries</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="detailsLabel">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>170</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 9pt &quot;Lucida Sans&quot;;</string>
    </property>
    <property name="plainText">
     <string>ROUTE_TIMEOUT = BASE_TIMER * 6
DELETE_TIMEOUT = BASE_TIMER * 4</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="timerSpinBox">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>160</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Lucida Sans&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="maxHopSpinBox">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>240</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Lucida Sans&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>15</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="logsLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Lucida Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Logs</string>
    </property>
   </widget>
   <widget class="QLabel" name="baseTimeLabel">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>130</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Lucida Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Base time</string>
    </property>
   </widget>
   <widget class="QLabel" name="HopLabel">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>210</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Lucida Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Max hop count</string>
    </property>
   </widget>
   <widget class="QLabel" name="routingLabel">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>290</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Lucida Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Routing table</string>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>0</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 36pt &quot;Lucida Sans&quot;;
text-decoration: underline;</string>
    </property>
    <property name="text">
     <string>RIP v2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1310</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
